xla/pjrt/gpu/BUILD

```diff
@@ -1,7 +1,8 @@
 load("@local_config_cuda//cuda:build_defs.bzl", "if_cuda")
 load("//xla:xla.bzl", "xla_cc_test")
-load("//xla/stream_executor:build_defs.bzl", "if_cuda_or_rocm", "if_gpu_is_configured")
+load("//xla/stream_executor:build_defs.bzl", "if_gpu_is_configured")
 load("@local_config_rocm//rocm:build_defs.bzl", "if_rocm")
+load("@tsl//tsl:tsl.bzl", "if_cuda_or_rocm")
```

Comment:
Maybe also revert changes to this file. It seems that the if_cuda_or_rocm macro from stream_executor:build_defs.bzl is the same as if_gpu_is_configured (surprisingly). So changing this macro load is a behavior change.